lib/OrdinaryDiffEqRosenbrock/src/OrdinaryDiffEqRosenbrock.jl

```diff
@@ -48,15 +48,21 @@ function rosenbrock_wanner_docstring(description::String,
         extra_keyword_default = "",
         with_step_limiter = false)
     keyword_default = """
+    chunk_size = Val{0}(),
```

Comment:
@ChrisRackauckas Rosenbrock methods that lack a reference:
ROS34PW1a
ROS34PW2
ROS34PW3

Reply:
https://www.sciencedirect.com/science/article/pii/S0377042716303934 has a lot of references in table 4.  These particular methods are from https://link.springer.com/article/10.1007/s10543-005-0035-y